packages/cli/generators/app/templates/test/ping.controller.test.ts.ejs

```diff
@@ -1,6 +1,6 @@
 import {createClientForHandler, supertest} from '@loopback/testlab';
 import {RestServer} from '@loopback/rest';
-import {<%= project.applicationName %>} from '../';
```

Comment:
This is intentional. We import from the root to mimic the usage of a module. `../src` assumes the test will be transpiled into the same directory as src.

Reply:
That makes sense, I'll revert this change.